app/assets/javascripts/global.js

```diff
@@ -13,8 +13,14 @@ $(document).ready(function() {
     return false;
   });
 
-  $("#select_all").click(function() {
-    $(".select-all-box").prop('checked', this.checked);
+  $('.check-all').on('change', function() {
+    // The check-all's checkbox value will contain the name of the checkboxes which are meant to be checked
+    $(`.checkbox[name="${this.value}"]`).prop('checked', this.checked);
```

Comment:
I wonder if this (when selected) impacts the data we send to the server. Typically input `name`s are the key for the data we send in a form, so does this add an extra `marked_ids[]=true` to the submitted form? IMO this would be safer as a `data` key

Reply:
Oh. This is the _value_, huh. Does this checkbox have a name in the final HTML?